### DISCUSSION TITLE
Polls and Tasks linking to current page

### DISCUSSION THREAD

**Discussion by DrakeHamString** [nextcloud/all-in-one#4936] [Questions] [answered]
### Steps to reproduce
1. Open Nextcloud
2. Click on "Polls" or "Tasks"

### Expected behavior 
Opening Polls or Tasks

### Actual behavior 
Links to current page

### Host OS 
SUSE Linux Enterprise Server 15 SP5

#### Nextcloud AIO version 
Nextcloud AIO v9.1.0

#### Current channel 
latest

#### Other valuable info 
When being in Files you can see that the buttons of Polls and Tasks point to Files again. Reinstalling them did not solve the issue
Here is a screenshot. (There you can see German text - Umfragen > Polls, Aufgaben > Tasks)
![image](https://github.com/nextcloud/all-in-one/assets/20707712/afed4ba9-8027-41d4-b34f-29f3ea1187d3)

**Comment by szaimen** [comment, marked as answer]
Duplicate of https://github.com/nextcloud/fulltextsearch/issues/842
